src/Matcher/ResponseContentMatcher.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Lakion\ApiTestCase\Matcher;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class ResponseContentMatcher implements ResponseContentMatcherInterface
+{
+    /**
+     * @var MatcherInterface
+     */
+    private $matcher;
+
+    /**
+     * @var string
+     */
+    private $error;
+
+    /**
+     * @var string
+     */
+    private $difference;
+
+    /**
+     * @param MatcherInterface $matcher
+     */
+    public function __construct(MatcherInterface $matcher)
+    {
+        $this->matcher = $matcher;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function matchResponse($actualResponse, $expectedResponse)
```

Comment:
@lchrusciel :+1:, then we can merge `MatcherInterface` and `ResponseContentMatcherInterface` into one, as they do the same job.